models/spleen_deepedit_annotation/configs/inference.json

```diff
@@ -5,7 +5,12 @@
         "$import ignite"
     ],
     "bundle_root": ".",
+    "image_key": "image",
     "output_dir": "$@bundle_root + '/eval'",
+    "output_ext": ".nrrd",
```

Comment:
HI @mingxin-zheng , I suggest to set default values on these args, otherwise the bundle output will be different.